### DISCUSSION TITLE
Replacing Diksha Logo in sunbirdEd-portal 4.10

### DISCUSSION THREAD

**Discussion by sonaliTekdi** [Sunbird-Ed/Community#744] [General Discussion]
@rajeevsathish I would appreciate it if I could find a solution from you regarding the issue below.

We are currently using version 4.10 of the sunbirdEd-portal. While launching the portal, we have noticed that the Diksha logo appears in various places. We want to replace it with a logo that is specific to our needs.

**What We Have Tried:**
- We attempted to replace the Sunbird logo in the assets folder, but it didn't worked.
[https://github.com/NIUANULP/nulp-portal/tree/elite-deployment-test/src/app/client/src/assets/images](https://github.com/NIUANULP/nulp-portal/tree/elite-deployment-test/src/app/client/src/assets/images)

- Commenting out the logo routes in the relevant files resulted in the old Sunbird logo appearing in the content editors.
[https://github.com/NIUANULP/nulp-portal/blob/elite-deployment-test/src/app/helpers/tenantHelper.js#L92](https://github.com/NIUANULP/nulp-portal/blob/elite-deployment-test/src/app/helpers/tenantHelper.js#L92)

![Screenshot from 2024-09-02 19-42-50](https://github.com/user-attachments/assets/f8b51371-f615-41af-a30f-3cafd10714b7)

**Request for Help:**
Could someone please guide us on how to successfully replace the Diksha logo with our custom logo across the portal?

cc : @komalm  @paritshivani

**Comment by HarishGangula** [comment]
@sonaliTekdi  If look at the code https://github.com/NIUANULP/nulp-portal/blob/elite-deployment-test/src/app/helpers/tenantHelper.js#L92
The assets and images are loaded from the tenant folderhttps://github.com/NIUANULP/nulp-portal/blob/elite-deployment-test/src/app/tenant/sunbird/images/appLogo.png  like this.

Check the api call tenant info what is the tenant is called if that is not present it defaults to default tenant

**Reply by sonaliTekdi** [reply]
Thank you for the detailed explanation!  Appreciate your help!
